src/com/projectkorra/projectkorra/event/AbilityVelocityAffectEntityEvent.java

```diff
@@ -0,0 +1,75 @@
+package com.projectkorra.projectkorra.event;
+
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.util.Vector;
+
+import com.projectkorra.projectkorra.ability.Ability;
+
+/**
+ * Cancellable event called when an ability would push or alter the velocity of
+ * an entity.
+ * 
+ * the entity can be changed, vector can be modified, and the ability that
+ * caused the change can be accessed.
+ *
+ * @author dNiym
+ *
+ */
+
+public class AbilityVelocityAffectEntityEvent extends Event implements Cancellable {
+
+    Entity affected;
+    Vector newVector;
```

Comment:
Could you please change the name of this private instance variable to something more meaningful like velocity or Velocity